### PR TITLE
docs/fix: readme and action

### PR DESCRIPTION
**Pull request by crondinini**
### Description

Changes in this PR:
- Removed projects that no longer existed from both README ~and CODEOWNERS~
~- Fixed the Github handle associated with projects that I know the Applications and Payment team owns (let me know if I've missed any)~
- Modified the github action that makes issues as stale to go over issues/PR in ascending order + increased the [`operations-per-run` limit](https://github.com/actions/stale#operations-per-run)

### CODE REVIEW

**Comment by aaronmgdr** [comment]
there's an open PR to update code owners https://github.com/celo-org/celo-monorepo/pull/8279

**Comment by aaronmgdr** [comment]
Thanks for all this cleanup!

**Comment by crondinini** [comment]
Thanks @aaronmgdr, I'll have a look and merge the 2.

File: README.md [hunk 1]
```diff
@@ -107,23 +107,16 @@ See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a
 The repository has the following packages (sub projects):
 
 - [attestation-service](packages/attestation-service) - service run by validators on the Celo network to send SMS messages, enabling attestations of user phone numbers and their accounts on the Celo network
-- [blockchain-api](packages/blockchain-api) - service that uses Blockscout to present view of transactions by account for Celo Wallet activity feed
 - [celotool](packages/celotool) - scripts for deploying and managing testnets
 - [cli](packages/cli) - tool that uses ContractKit to interact with the Celo protocol ([docs](https://docs.celo.org/command-line-interface/introduction))
-- [contractkit](packages/contractkit) - library to help developers and validators interact with the protocol and it's smart contracts ([docs](https://docs.celo.org/celo-sdk/contractkit))
```
Comment (aaronmgdr) [inline README.md:113]: this is inside packages/sdk so lets keep and move the link there

Comment (aaronmgdr) [inline README.md:113]: this is inside packages/sdk so lets keep and move the link there

File: README.md [hunk 2]
```diff
@@ -107,23 +107,16 @@ See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a
...
 - [helm-charts](packages/helm-charts) - templatized deployments of entire environments to Kubernetes clusters
-- [mobile](packages/mobile) - Android wallet app for the Celo platform ([docs](https://docs.celo.org/getting-started/using-the-mobile-wallet), [live](https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores))
-- [notification-service](packages/notification-service) - service for managing push notifications for Celo Wallet
 - [protocol](packages/protocol) - identity, stability and other smart contracts for the Celo protocol ([docs](https://docs.celo.org/celo-codebase/protocol))
-- [react-components](packages/react-components) - generalized react native components
-- [reserve-site](packages/reserve-site) - website for ([Celo Reserve.org](https://celoreserve.org/))
+- [sdk](packages/sdk)
 - [terraform-modules](packages/terraform-modules) - templatized deployments of entire VM-based testnets for Google Cloud Platform
 - [typescript](packages/typescript) - no README available (improve?)
-- [utils](packages/utils) - no README available (improve?)
```
Comment (aaronmgdr) [inline README.md:126]: exists inside sdk

Comment (aaronmgdr) [inline README.md:126]: exists inside sdk

File: README.md (hunk 1 above)
Comment (crondinini) [reply README.md:113]: Added back

File: README.md (hunk 2 above)
Comment (crondinini) [reply README.md:126]: There's no explanation of what the utils are for. 
Considering it's inside the SDK, in my mind it there's no extra value in explicitly calling it out here. I'd vote to leave it be implicitly part of the SDK and not in this README list. Let me know if you disagree/what you think.

Comment (crondinini) [reply README.md:126]: On the topic of the SDK, could you help me come up with a short sentence of what the SDK includes so I can put in this README?

Comment (aaronmgdr) [reply README.md:126]: My first thought

`Typescript packages for interacting with Celo, including Contracts, Wallets, Crypto ++`

File: README.md [hunk 3]
```diff
@@ -107,23 +107,17 @@ See the [issue backlog](https://github.com/celo-org/celo-monorepo/issues) for a
...
 - [faucet](packages/faucet) - faucet deployment configuration ([live](https://celo.org/build/faucet))
 - [helm-charts](packages/helm-charts) - templatized deployments of entire environments to Kubernetes clusters
-- [mobile](packages/mobile) - Android wallet app for the Celo platform ([docs](https://docs.celo.org/getting-started/using-the-mobile-wallet), [live](https://play.google.com/store/apps/details?id=org.celo.mobile.alfajores))
-- [notification-service](packages/notification-service) - service for managing push notifications for Celo Wallet
 - [protocol](packages/protocol) - identity, stability and other smart contracts for the Celo protocol ([docs](https://docs.celo.org/celo-codebase/protocol))
-- [react-components](packages/react-components) - generalized react native components
-- [reserve-site](packages/reserve-site) - website for ([Celo Reserve.org](https://celoreserve.org/))
+- [sdk](packages/sdk)
+- [contractkit](packages/sdk/contractkit) - library to help developers and validators interact with the protocol and its smart contracts ([docs](https://docs.celo.org/celo-sdk/contractkit))
```
Comment (aaronmgdr) [inline README.md]: Oh i just noticed that link  for docs is broken

https://docs.celo.org/developer-guide/contractkit

really a good thing you are bringing our attention to this

Comment (crondinini) [reply README.md]: Fixed!

**Review by aaronmgdr — APPROVED** [review]

**Review by SimsonEnduro — APPROVED** [review]

**Review by SimsonEnduro — APPROVED** [review]

**Pull request merged**